### PR TITLE
Inventory UI

### PR DESCRIPTION
**Pull request by AwesomestCode**
Implements the survival inventory in Prismarine Web Client

Progress:
- [x] Displays your log-in inventory.
- [x] Dynamically updates the inventory when it's changed
- [x] Allows you to change the inventory on Prismarine Web Client
- [ ] Styling

### CODE REVIEW

File: lib/menus/utils/image-util.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+  if (IMAGE_ROOT.includes('.com')) {path = path.replace('block/', 'blocks/'); path=path.replace('item/', 'items/')}
+  //if (path.includes('enchant_table_anim'))path='enchant_table_anims2'
+  console.log(path)
+  return path
+}
+
+window.loadedImageBlobs = {}
+
+const images = [
+  'item/brick',
```
Comment (rom1504) [inline lib/menus/utils/image-util.js:23]: Why is this hard coded?

Comment (AwesomestCode) [reply lib/menus/utils/image-util.js]: That was left over from when those items were the only ones supported by the library.

**Comment by AwesomestCode** [comment]
Ready for review—not completely finished and has some bugs, but is mostly done. *Note: crafting won't be implemented in this PR because of how different it is.*

File: package.json [hunk 2]
```diff
@@ -37,6 +37,7 @@
     "compression": "^1.7.4",
     "express": "^4.17.1",
     "lit": "^2.0.2",
+    "minecraft-inventory-gui": "github:AwesomestCode/prismarine-inventory-gui",
```
Comment (rom1504) [inline package.json:40]: can you instead depend on the released version of this ?

Comment (AwesomestCode) [reply package.json:40]: No; the released version doesn’t have all the features I need.

Comment (rom1504) [reply package.json:40]: ok, then PR the features you need upstream
then you will be able to use the released version here

Comment (bobisnumber123) [reply package.json:40]: sorry for asking but what does PR mean?

Comment (Hedgie1) [reply package.json:40]: pull request

File: lib/menus/utils/image-util.js [hunk 3]
```diff
@@ -0,0 +1,102 @@
+import { layouts } from 'minecraft-inventory-gui/lib/layouts.mjs'
```
Comment (extremeheat) [inline lib/menus/utils/image-util.js:1]: This file can be refactored for use the image data already bundled inside the client.

**Review by WindowsRefundDay — APPROVED** [review]

**Comment by WindowsRefundDay** [comment]
@rom1504

**Comment by rom1504** [comment]
Do you have any questions on how to finish this PR @WindowsRefundDay ?

File: .gitignore [hunk 4]
```diff
@@ -1,5 +1,4 @@
 node_modules/
-package-lock.json
```
Comment (rom1504) [inline .gitignore:2]: please add this back

File: package.json (hunk 2 above)
Comment (rom1504) [reply package.json:41]: this still need to be updated to the npm version of minecraft-inventory-gui ; so you'd need to PR any change you made to the repo there

File: .gitignore (hunk 4 above)
Comment (AwesomestCode) [reply .gitignore:2]: Sorry looks like that happened when I rebased it off my Mineflayer bump PR a week ago; fixed

**Comment by rom1504** [comment]
can you delete package-lock.json ?

**Review by Pixel556 — COMMENTED** [review]
Inventory would be nice!

**Comment by AwesomestCode** [comment]
Hotbar is already implemented?

**Comment by rom1504** [comment]
old, feel free to re open if you want to finish it

**Pull request closed without merging**